css/properties/align-content.json

```diff
@@ -6,6 +6,7 @@
           "__compat": {
             "description": "Supported in Flex Layout",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/align-content",
+            "spec_url": "https://drafts.csswg.org/css-flexbox-1/#align-content-property",
```

Comment:
This raises the question of whether we should make sure to add all spec URLs for a single CSS property at once, which is incompatible with doing one spec at a time. In https://github.com/w3c/browser-compat-data/commit/4f66425be4e95a40e5f43ec5c02c2681ce337316 this also links to https://drafts.csswg.org/css-align-3/#propdef-align-content but I trimmed it to just this Flexbox URL.

Reply:
Having both specs from scraping would be fine, imo.

Reply:
```suggestion
            "spec_url": [
              "https://drafts.csswg.org/css-align/#propdef-align-content",
              "https://drafts.csswg.org/css-flexbox/#propdef-align-content"
            ],
```